spec/fixtures/vcr_cassettes/sales_invoice_all.yml

```diff
@@ -2,7 +2,7 @@
 http_interactions:
 - request:
     method: get
-    uri: https://test.ucetnictvi.bonobo.cz/api/v1/sales_invoices/
+    uri: http://localhost:3339/api/v1/sales_invoices/
```

Comment:
By now I have no idea how to solve it. Old tests accesses to https://test.ucetnictvi.bonobo.cz, new -- to localhost. Either change everyone and make single config for pina, or revert cassettes and make per-test config.